packages/@okta/vuepress-site/docs/guides/custom-widget-gen3/main/index.md

```diff
@@ -77,6 +77,100 @@ See [Customization examples](#customization-examples) for snippets that you can
 
    > **Note:** To discard your changes without publishing them, click **Revert changes** or click the **Code editor** toggle again. Turning off the code editor restores the default JavaScript code.
 
+### Use afterRender with the third generation
+
+The third generation of the Sign-In Widget is built on [Preact](https://preactjs.com/), a lightweight React alternative. This means that the `afterRender` function doesn't work with the third generation.
```

Comment:
The `afterRender` function is still triggered, and useful for logging and "pure" callbacks. They just won't work with DOM manipulation and similar render-related changes.

```suggestion
The third generation of the Sign-In Widget is built on [Preact](https://preactjs.com/), a lightweight React alternative.
```